docs/www/events.html

```diff
@@ -1,9 +1,9 @@
 [%-
   page.import({
-      title    => 'Perl events',
+      title    => 'Perl and Raku events',
```

Comment:
Please drop 'and Raku' from the title and description.

Reply:
That was the agreed-upon language with the TPF president

Reply:
TPF doesn't run this site. I'm just asking for the page level content, not the event names.

Reply:
The events covered relate to both Perl and Raku events. Why would that be inappropriate?

Reply:
I'm going to remove 479020772a522f7260b204eb98ade471b23c0432 from this PR so we can get the critical stuff in. I'll open a separate PR for Raku issues.